FAQs/README.md

```diff
@@ -10,6 +10,7 @@
  2. [How to add new subscriptions or mangement groups after deploying AzTS?](#2-how-to-add-new-subscriptions-or-management-groups-after-deploying-azts)
  3. [On running the AzTS installation command (`Install-AzSKTenantSecuritySolution`) I am getting an error message "Tenant ID, application ID, principal ID, and scope are not allowed to be updated."](#3-on-running-the-azts-installation-command-install-azsktenantsecuritysolution-i-am-getting-an-error-message-tenant-id-application-id-principal-id-and-scope-are-not-allowed-to-be-updated)
  4. [While installing AzTS solution I have provided my preferences for telemetry collection i.e. anonymous AzTS usage data and organization/team contact details. How do I update my preferences now?](#4-while-installing-azts-solution-i-have-provided-my-preferences-for-telemetry-collection-ie-anonymous-azts-usage-data-and-organizationteam-contact-details-how-do-i-update-my-preferences-now)
+ 5. [On running the Autoupdater function getting some error related to function runtime version.](#5-on-running-the-autoupdater-function-getting-some-error-related-to-function-runtime-version)
```

Comment:
Could we pls. specify what error as "some error" is too ambiguous as a FAQ topic. Pls. take reference from point no. 3. 

Take for example: 
**On running the Autoupdater function (XYZ..) I am encountering an error related to conflicts in function runtime version**